src/components/BackButton/index.jsx

```diff
@@ -58,14 +58,12 @@ const MobileAwareBackButton = ({ onClick, to, arrow = false }) => {
   const router = useRouter()
   const { isMobile } = useBreakpoints()
   const location = router.getCurrentLocation()
-  if (!onClick && !to) {
-    to = location.pathname
-      .split('/')
-      .slice(0, -1)
-      .join('/')
+  let toToUse = to
```

Comment:
```suggestion
  const toToUse = to
```

Reply:
No, we can change its value just after https://github.com/cozy/cozy-banks/pull/2306/files/5c8b7407d7d35f9c70d8ef95db7f9985a683cc0e#diff-99412e8036265118649a2f7b2f814bcd5f979e9ff94971951e28185bc4c52c79R63